wavefront/provider.go

```diff
@@ -78,7 +78,48 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 
 }
 
-var wfMutexKV = mutexkv.NewMutexKV()
+// MutexKV is a simple key/value store for arbitrary mutexes. It can be used to
```

Comment:
This code should be put in its own file and commented as to its origin. In other words, best not to mix our code with HashiCorp code in the same file.

Reply:
Done.